src/pykiso/test_coordinator/test_case.py

```diff
@@ -213,7 +213,7 @@ def define_test_parameters(
     run_timeout: Optional[int] = None,
     teardown_timeout: Optional[int] = None,
     test_ids: Optional[dict] = None,
-    variant: Optional[dict] = None,
+    tag: Optional[dict] = None,
```

Comment:
I'd really go for more complete type hints
```suggestion
    tag: Optional[Dict[str, List[str]]] = None,
```

Reply:
Done